From 554ec54065c1ed6a715c192f5cf7b77aad6631dd Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Wed, 27 Sep 2023 09:27:39 +0200
Subject: [PATCH] Update com.webos.service.videooutput.role.json.in: Add
 trustLevel

Fixes:

Sep 27 07:51:47 qemux86-64 ls-hubd[290]: [] [pmlog] ls-hubd LSHUB_ROLE_FILE {"FILE":"file_parser.cpp","LINE":195} No trust level specified for application in role file (/usr/share/luna-service2/roles.d/app-shell.role.json)
---
Upstream-Status: Submitted

 files/sysbus/com.webos.service.videooutput.role.json.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/files/sysbus/com.webos.service.videooutput.role.json.in b/files/sysbus/com.webos.service.videooutput.role.json.in
index 2ac2e2f..e172c0b 100755
--- a/files/sysbus/com.webos.service.videooutput.role.json.in
+++ b/files/sysbus/com.webos.service.videooutput.role.json.in
@@ -1,5 +1,6 @@
 {
     "exeName":"@WEBOS_INSTALL_SBINDIR@/videooutputd",
+    "trustLevel": "oem",
     "type": "regular",
     "allowedNames": ["com.webos.service.videooutput"],
     "permissions": [
